docs/options.md

```diff
@@ -348,7 +351,10 @@ CIBW_DEPENDENCY_VERSIONS: ./constraints.txt
 ### `CIBW_TEST_COMMAND` {: #test-command}
 > Execute a shell command to test each built wheel
 
-Shell command to run tests after the build. The wheel will be installed automatically and available for import from the tests. `{project}` can be used as a placeholder for the absolute path to the project's root and will be replaced by `cibuildwheel`.
+Shell command to run tests after the build. The wheel will be installed automatically and available for import from the tests. To ensure the wheel is imported by your tests (instead of your source copy), tests are run from a different directory. Use the placeholders `{project}` and `{package}` when specifying paths in your project.
+
+- `{project}` is an absolute path to the project root - the working directory where cibuildwheel was called.
```

Comment:
Do we still want to advertise this?

Reply:
Before putting this PR together I thought we could have completely removed it. But it is still necessary in CIBW_TEST_COMMAND, because that command is run from a different cwd, so there still needs to be a way to reference something that's inside the project but outside the package.

Reply:
Ah, yes. I forgot the exact reasons, but fair enough, then we need to keep it :-/